CHANGELOG.rst

```diff
@@ -8,6 +8,31 @@ This project adheres to `Semantic Versioning <http://semver.org/>`_.
 
 .. contents:: ``Release Versions``
 
+`3.5.0`_
+=====================
+
+**Release Date:** ``2023-05-09``
+
+Added
+-----
+
+- The following new modules are included in this release
+    - ``dcnm_image_upgrade`` - Module for managing NDFC image upgrade
+    - ``dcnm_image_upload`` - Module for managing NDFC image upload
+    - ``dcnm_image_policy`` - Module for managing NDFC image policy
+    - ``dcnm_vpc_pair`` - Module for managing dcnm NDFC switch pairs
```

Comment:
I propose a slight update to the wording here:

` Module for managing dcnm NDFC vPC switch pairs`